DFC.FindACourseClient/Repositories/CosmosRepository.cs

```diff
@@ -27,12 +27,63 @@ public CosmosRepository(CourseSearchAuditCosmosDbSettings cosmosDbConnection, ID
 
         public async Task<HttpStatusCode> UpsertAsync(T model)
         {
-            var ac = new AccessCondition { Condition = model.Etag, Type = AccessConditionType.IfMatch };
-            var pk = new PartitionKey(model.PartitionKey);
-
-            var result = await documentClient.UpsertDocumentAsync(DocumentCollectionUri, model, new RequestOptions { AccessCondition = ac, PartitionKey = pk }).ConfigureAwait(false);
+            var accessCondition = new AccessCondition { Condition = model.Etag, Type = AccessConditionType.IfMatch };
+            var partitionKey = new PartitionKey(model.PartitionKey);
+            var result = await documentClient.UpsertDocumentAsync(DocumentCollectionUri, model, new RequestOptions { AccessCondition = accessCondition, PartitionKey = partitionKey }).ConfigureAwait(false);
 
             return result.StatusCode;
         }
+
+        public async Task InitialiseDatabaseAsync(bool isDevelopment)
```

Comment:
parameter `isDevelopment` is not used, remove if not required

Reply:
Made use of the parameter